### PR TITLE
Illumio Сore data connector PR

### PR DESCRIPTION
**Pull request by spsocprime**
Change(s):
   - Added data connector

### CODE REVIEW

File: Solutions/Illumio Core/Data Connectors/Connector_IllumioCore_CEF.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
+{
+    "id": "IllumioCore",
+    "title": "Illumio Core",
+    "publisher": "Illumio",
+    "descriptionMarkdown": "The [Illumio Core](https://www.illumio.com/products/core) data connector provides the capability to ingest Illumio Core logs into Microsoft Sentinel.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**IllumioCoreEvent**](https://aka.ms/sentinel-IllumioCore-parser) which is deployed with the Microsoft Sentinel Solution.",
```
Comment (rushriva) [inline Solutions/Illumio Core/Data Connectors/Connector_IllumioCore_CEF.json:6]: https://aka.ms/sentinel-IllumioCore-parser - link does not exist

File: Sample Data/CEF/IllumioCore.json [hunk 2]
```diff
@@ -0,0 +1,1542 @@
+[
```
Comment (rushriva) [inline Sample Data/CEF/IllumioCore.json:1]: Sample data is not in RAW(CEF) format.

File: Solutions/Illumio Core/Data Connectors/Connector_IllumioCore_CEF.json (hunk 1 above)
Comment (spsocprime) [reply Solutions/Illumio Core/Data Connectors/Connector_IllumioCore_CEF.json:6]: Such links are created by Microsoft team

File: Sample Data/CEF/IllumioCore.json (hunk 2 above)
Comment (spsocprime) [reply Sample Data/CEF/IllumioCore.json]: fixed

**Review by rushriva — APPROVED** [review]
Approved, after comments are addressed

**Pull request merged**